src/Control/PaginationControl.php

```diff
@@ -496,6 +496,17 @@ protected function createPageSelectorItem()
     protected function assemble()
     {
         if ($this->getPageCount() < 2) {
+            if ($this->getCurrentPageNumber() > 1) {
+                $this->add(Html::tag(
```

Comment:
I would suggest to use `addHtml()` instead of `add()`.
Please use the `ipl\Web\Widget\Link` class instead of creating `Html::tag('a'...)`